src/main/java/core/basesyntax/service/impl/FilesParserImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FilesParser;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FilesParserImpl implements FilesParser {
+    private static final String SEPARATOR = ",";
+    private static final int INDEX_ZERO = 0;
+    private static final int INDEX_ONE = 1;
+    private static final int INDEX_TWO = 2;
```

Comment:
Constants names should describe meaning but not value itself
```suggestion
    private static final int OPERATION_TYPE_INDEX = 0;
    private static final int FRUIT_NAME_INDEX = 1;
    private static final int FRUIT_QUANTITY_INDEX = 2;
```